util-hadoop/src/main/java/com/google/cloud/hadoop/util/CredentialFromAccessTokenProviderClassFactory.java

```diff
@@ -80,22 +79,9 @@ protected TokenResponse executeRefreshToken() throws IOException {
 
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
 
-  /** Generate the credential from the {@link AccessTokenProvider}. */
-  public static Credential credential(
```

Comment:
What these changes are about?

Reply:
Extracting the logic to pull AccessTokenProvider out of this class since it's used in a larger scope.